Description: Fix old meson version bugs
 The old version of meson present in Ubuntu 18.04 doesn't support various
 things, so we remove them here.

--- gbcc-0.1.GBCC_GIT_REVISION.orig/meson.build	2020-01-19 15:17:10.345531949 +0000
+++ gbcc-0.1.GBCC_GIT_REVISION/meson.build	2020-01-19 15:11:28.000000000 +0000
@@ -4,7 +4,6 @@
   license: 'MIT',
   default_options: [
     'c_std=c11',
-    'optimization=3',
     'buildtype=release',
     'warning_level=3',
     'b_lto=true',
@@ -32,22 +31,22 @@
 endif
 
 tileset_location = join_paths(
-  debug ? '' : data_location,
+  data_location,
   'tileset.png'
 )
 
 printer_sound_location = join_paths(
-  debug ? '' : data_location,
+  data_location,
   'print.wav'
 )
 
 gtk_ui_location = join_paths(
-  debug ? 'src/gtk' : data_location,
+  data_location,
   'gbcc.ui'
 )
 
 shader_location = join_paths(
-  debug ? '' : data_location,
+  data_location,
   'shaders',
   ''
 )

--- gbcc-0.1.GBCC_GIT_REVISION.orig/meson_options.txt	2020-01-19 15:11:56.907926883 +0000
+++ gbcc-0.1.GBCC_GIT_REVISION/meson_options.txt	2020-01-19 15:12:15.307316438 +0000
@@ -1,2 +1,2 @@
-option('man-pages', type: 'feature', value: 'auto', description: 'Install man pages.')
-option('gtk', type: 'feature', value: 'auto', description: 'Build & install the GTK GUI')
+option('man-pages', type: 'boolean', value: false, description: 'Install man pages.')
+option('gtk', type: 'boolean', value: true, description: 'Build & install the GTK GUI')

